openmc/mgxs/mgxs.py

```diff
@@ -252,7 +252,7 @@ def __deepcopy__(self, memo):
             clone._name = self.name
             clone._rxn_type = self.rxn_type
             clone._by_nuclide = self.by_nuclide
-            clone._nuclides = copy.deepcopy(self._nuclides)
+            clone._nuclides = copy.deepcopy(self._nuclides, memo)
```

Comment:
Is this necessary in the event that `_nuclides` is a list of `Nulicde` objects instead of strings?